includes/Story_Post_Type.php

```diff
@@ -178,6 +180,20 @@ public function skip_amp( $skipped, $post ) {
 		return $skipped;
 	}
 
+	/**
```

Comment:
Isn't this a repeated here https://github.com/google/web-stories-wp/pull/3621

Reply:
@spacedmonkey Sort of. There's some separate internal discussion going on about AMP compatibility. Best to hold off on reviewing these two.